flow-typed/platform.js

```diff
@@ -143,3 +143,23 @@ declare type CookieOpts = {
 	isSecure?: boolean,
 	encoding?: string,
 };
+
+// See https://docs.launchdarkly.com/docs/node-sdk-reference#section-users
+declare type LaunchDarklyUser$CustomAttributes = {
+	RequestCountry?: string,
+	RequestRegion?: string,
```

Comment:
LaunchDarkly allows any custom attribute names, but I decided to reserve these two as fields in order to enforce naming convention (the same property names are used by Chapstick when passing resolved geo location to LaunchDarkly)